### ISSUE TITLE
Missing events in admin logs

### ISSUE THREAD

**Issue by SilasPeters** [svsticky/constipated-koala#1071]
As explained in #658, the logs in the admin page, which displays events like "person x enrolled for activity y", does not include all events.

#658 suggests getting a new logger, so these issues go hand-in-hand, as getting a new logger might just as well solve it.

**Comment by Riscky** [comment]
I don't think that using a new logging library is going to magically solve the missing events. The logs are configured with the magic `is_impressionable` attribute on models and overwriting the `impressions` object , and I suppose we'll have to add that everywhere we want logs.
